Maple2.Server.Game/Model/Field/Actor/FieldPlayer.cs

```diff
@@ -70,10 +83,14 @@ public class FieldPlayer : Actor<Player> {
             InBattle = false;
         }
 
-        Buffs.Update(tickCount);
+        base.Update(tickCount);
```

Comment:
Doesn't this happen already?

Reply:
I replaced Buffs.update with base.update so that more shared systems could be put into base & we won't have to worry about copying the logic across both.